lib/ui/mockData/MockUserProfileViewModel.dart

```diff
@@ -0,0 +1,80 @@
+import 'dart:math';
+
+import 'package:farmsmart_flutter/ui/profile/UserProfile.dart';
+import 'package:farmsmart_flutter/ui/profile/UserProfileListItem.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:farmsmart_flutter/data/model/mock/MockString.dart';
+
+class MockUserProfileViewModel {
+  static UserProfileViewModel build() {
```

Comment:
add another build with large titles and add it to playground

Reply:
Done